src/Microsoft.Graph.Auth/ConfidentialClient/AuthorizationCodeProvider.cs

```diff
@@ -44,13 +44,29 @@ public class AuthorizationCodeProvider : MsalAuthenticationBase, IAuthentication
         /// <param name="tenant">Tenant to sign-in users. This defaults to <c>common</c> if non is specified.</param>
         /// <param name="nationalCloud">A <see cref="NationalCloud"/> which identifies the national cloud endpoint to use as the authority. This defaults to the global cloud <see cref="NationalCloud.Global"/> (https://login.microsoftonline.com).</param>
         /// <returns>A <see cref="IConfidentialClientApplication"/></returns>
+        /// <exception cref="AuthenticationException"></exception>
```

Comment:
Missing explanation of when the exception is thrown. Customers aren't always looking at our source :-)

Reply:
True for all *provider comments.